src/CMakeLists.txt

```
@@ -1,4 +1,4 @@
-add_library(
+set(SOURCE_FILES
     inexor-vulkan-renderer
 
     vulkan-renderer/application.cpp
```

```suggestion
set(SOURCE_FILES
    vulkan-renderer/application.cpp
```
Sorry, forgot to add this bit.

---

Done.